orte/mca/ess/hnp/ess_hnp_module.c

```diff
@@ -621,12 +623,12 @@ static int rte_init(void)
         goto error;
     }
     /* set the event base */
-    opal_pmix_base_set_evbase(orte_event_base);
+   opal_pmix_base_set_evbase(orte_event_base);
 
     /* setup the routed info - the selected routed component
      * will know what to do.
      */
-    if (ORTE_SUCCESS != (ret = orte_routed.init_routes(ORTE_PROC_MY_NAME->jobid, NULL))) {
+     if (ORTE_SUCCESS != (ret = orte_routed.init_routes(ORTE_PROC_MY_NAME->jobid, NULL))) {
```

Comment:
This is a nit pick: none of the whitespace changes in this file seemed necessary, and actually seem incorrect (albeit harmless).